src/app/Components/loggedIn.jsx

```diff
@@ -34,6 +27,7 @@ var LoggedIn = React.createClass({
         this.props.lock.logout({ref: 'window.location.href'});
       }
       this.props.dispatch(getUser(profile));
+      this.navToFriends();
```

Comment:
Automatically route to friends after the component mounts...

We may want to only call this in the success callback after fetching the appFriends from FB